src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs

```diff
@@ -43,6 +43,8 @@ public class Settings : ISettings
 
         private readonly Dictionary<string, VirtualSettingSection> _computedSections;
 
+        public Dictionary<string, VirtualSettingSection> ComputedSections { get { return _computedSections; } }
```

Comment:
```suggestion
        public Dictionary<string, VirtualSettingSection> ComputedSections { get; }
```

Since the getter does nothing more than return a private field, we can let the compiler do this automatically for us with an "auto prop". You'll need to change all references of `_compuedSections` to `ComputedSections`, but otherwise it works the same. (well, almost the same, if you want to be pedantic?  Technically the IL will have an additional method call, the getter function, but the JIT might inline/optimize it away anyway, so real runtime perf probably isn't affected)

This is what @donnie-msft  suggested in https://github.com/NuGet/NuGet.Client/pull/5126#discussion_r1167154519, but I think you misunderstood his suggestion.